TrackletAlgorithm/TrackletProcessor.h

```diff
@@ -316,10 +360,10 @@ TC::barrelSeeding(const AllStub<InnerRegion> &innerStub, const AllStub<OuterRegi
 // impact parameter.
 
   bool valid_rinv=abs(*rinv) < floatToInt(rinvcut, krinv);
-  bool valid_z0=abs(*z0) < ((Seed == TF::L1L2) ? floatToInt(z0cut, kz0) : floatToInt(1.5 * z0cut, kz0));
+  bool valid_z0=abs(*z0) < ((Seed == TF::L1L2) ? floatToInt(z0cut, kz0) : floatToInt(1.5*z0cut,kz0));
 
-  const ap_int<TrackletParameters::kTParPhi0Size + 2> phicrit = *phi0 - (*rinv<<1);
-  const bool keep = (phicrit > 9253) && (phicrit < 56269);
+  const ap_int<TrackletParameters::kTParPhi0Size + 2> phicrit = *phi0 - (*rinv>>8)*469;//fixme, calculation adjusted to match emulation
+  const bool keep = (phicrit > 10190) && (phicrit < 55345);
```

Comment:
@jasonfan393 Could you please eliminate the magic numbers (469, 10190, and 55345) by defining meaningfully named constants in Constants.h? I think you can follow the same naming scheme as is used in the emulation.